proto/detectors.proto

```diff
@@ -977,6 +977,17 @@ enum DetectorType {
   Ngrok = 966;
   Replicate = 967;
   Postgres = 968;
+  AzureActiveDirectoryApplicationSecret = 969;
+  AzureCacheForRedisAccessKey = 970;
+  AzureCosmosdbKeyIdentifiable = 971;
```

Comment:
nit: I think we should probably make the `db` capitalized so it's similar to other detectors with the same name. (ex `MongoDB`
`AzureCosmosdbKeyIdentifiable` - >  `AzureCosmosDBKeyIdentifiable`